app/helpers/spotlight/application_helper.rb

```diff
@@ -136,16 +136,35 @@ def available_view_fields
     end
 
     def exhibit_stylesheet_link_tag(tag)
-      if current_exhibit_theme && current_exhibit.theme != 'default'
-        stylesheet_link_tag "#{tag}_#{current_exhibit_theme}"
+      Deprecation.warn 'exhibit_stylesheet_link_tag has been deprecated and will be removed.  '\
```

Comment:
I've noticed that these deprecation warnings are not using the supplied message. I think we may need to pass an instance of `self` to them.

```ruby
      Deprecation.warn self, 'exhibit_stylesheet_link_tag has been deprecated and will be removed.  '\
```

Could we also pass the message that this will be removed in Spotlight 3.0?